pkg/controllers/deployment/default_deployment.go

```diff
@@ -84,12 +85,36 @@ func getOAuthServerDeployment(
 	templateSpec.Volumes = append(templateSpec.Volumes, v...)
 	container.VolumeMounts = append(container.VolumeMounts, m...)
 
+	argsRaw, err := observeoauth.GetOAuthServerArgumentsRaw(&operatorConfig.Spec.ObservedConfig)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to get audit sync data: %w", err)
+	}
+
+	args, err := arguments.Parse(argsRaw)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse raw server arguments: %w", err)
+	}
+
+	klog.Infof("xxx serverArguments: %s", args)
```

Comment:
remove

Reply:
yes, I will `ripgrep | sed` all the xxx out of the codebase. thx for the reminder